subjects/maxwordcountn/README.md

```diff
@@ -0,0 +1,41 @@
+## maxwordcountn
+
+### Instructions
+
+Write a function `MaxWordCountN` that will return a `map` of the `n` words that occurs the most in a string `text`. This map will have the word as key and the number of occurences of this word as value.
```

Comment:
Maybe words should be explicitly defined, that they are separated by spaces

Reply:
Good idea ! I have added that thx